docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Sphinx >= 6.2.1
```

Comment:
I would move these to the Dockerfile.

Reply:
I am not sure what's the best strategy here. 
See the previous comment:
https://github.com/bdaiinstitute/spot_ros2/pull/93/files/549ac784ed49335e981fb7eaa7493c9bc5ecda9a#r1240099455

Reply:
Replied in the other thread.